settings/src/lib.rs

```diff
@@ -72,6 +72,8 @@ pub struct NetworkSettings {
     pub own_ip: IpAddr,
     /// Mesh IP of bounty hunter (in fd00::/8)
     pub bounty_ip: IpAddr,
+    /// Broadcast ip address used for peer discovery (in ff02::/8)
+    pub discovery_ip: Ipv6Addr,
```

Comment:
we need a default function and serde attribute if we want to remain config compatible

Reply:
done